public/index.html

```diff
@@ -6,12 +6,21 @@
     <meta name="theme-color" content="#000000">
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+
+        <!-- Compiled and minified CSS -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/css/materialize.min.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="preload" as="style" onload="this.rel='stylesheet'">
+
     <title>Authors Haven</title>
   </head>
   <body>
-    <noscript>
-      You need to enable JavaScript to run this app.
-    </noscript>
-    <div id="root"></div>
+
+  <div id="root"></div>
+
+    <!--Import jQuery before materialize.min.js-->
```

Comment:
Why do we need jquery ?

Reply:
oh, I was using it for some other designs on the page but since they are not included, I only need the icons. let me refactor. thank you